astropy/wcs/wcsapi/tests/test_fitswcs.py

```diff
@@ -30,7 +30,7 @@
 from astropy.utils import iers
 from astropy.utils.data import get_pkg_data_filename
 from astropy.utils.exceptions import AstropyDeprecationWarning, AstropyUserWarning
-from astropy.wcs.wcs import WCS, FITSFixedWarning, NoConvergence, Sip, WCSLIB_VERSION
+from astropy.wcs.wcs import WCS, WCSLIB_VERSION, FITSFixedWarning, NoConvergence, Sip
```

Comment:
should imports be sorted as well? I actually don't know the answer but I have a bout of OCD

Reply:
What's the problem here? As far as I can tell everything is working as intended.

Reply:
"W" before "F"